src/modules/home/communityBenefits/CommunityBenefitCard.tsx

```diff
@@ -1,21 +1,23 @@
 import Image from "next/image";
 import { CommunityBenefitCardProps } from "./helper";
+import { Card, CardContent, CardFooter } from "@/components/ui/card";
 
 export const CommunityBenefitCard = ({
   name,
   imageUrl,
 }: CommunityBenefitCardProps) => {
   return (
-    <div
-      key={name}
-      className="flex flex-col items-center rounded-xl border-2 bg-blue-100 p-8 shadow-xl"
-    >
-      <div className="flex flex-col items-center gap-4 font-semibold">
-        <div className="aspect-w-1 aspect-h-1 relative flex h-28 w-28 justify-center md:h-[10rem] md:w-[10rem]">
-          <Image src={imageUrl} fill alt={name} className="object-contain" />
-        </div>
-        <div className="mt-2 text-center text-lg">{name}</div>
-      </div>
-    </div>
+    <>
+      <Card className="flex flex-col items-center rounded-xl border-2 bg-blue-100 p-8 shadow-xl">
```

Comment:
same thing here,
https://github.com/xplorer-io/website/pull/101/files#r1845253529

apply the styles in the base component, and only add or remove or put custom styling only if we need to match the design. Also, if we need to apply different stylings, it inconsistent desgin. we need to reconsider the design  in that case.